essentials-framework/Essentials Core/PepperDashEssentialsBase/Routing/RoutingInterfaces.cs

```diff
@@ -89,6 +89,15 @@ public interface ITxRouting : IRouting
         IntFeedback VideoSourceNumericFeedback { get; }
         IntFeedback AudioSourceNumericFeedback { get; }
         void ExecuteNumericSwitch(ushort input, ushort output, eRoutingSignalType type);
+    }
+
+    /// <summary>
+    /// Defines a receiver that has internal routing (DM-RMC-4K-Z-SCALER-C)
+    /// </summary>
+    public interface IRmcRouting : IRouting
+    {
+        IntFeedback AudioVideoSourceNumericFeedback { get; }
+        void ExecuteNumericSwitch(ushort input, ushort output, eRoutingSignalType type);

```

Comment:
Does IRouting already specify this method? If so, no need to add it here. The existing method will be picked up already.

Reply:
No, it does not, it has ExecuteSwitch - ExecuteNumericSwitch is specifically added to ease bridging by using the simpl-defined numbers for switching - there's a similar mechanism in the ITxRouting interface.

Reply:
ah, got it. Didn't notice the naming difference.